README.md

```diff
@@ -16,84 +16,130 @@ pgvecto.rs is a Postgres extension that provides vector similarity search functi
 - 🦀 **Rewrite in Rust**: Rewriting in Rust offers benefits such as improved memory safety, better performance, and reduced **maintenance costs** over time.
 - 🙋 **Community**: People loves Rust We are happy to help you with any questions you may have. You could join our [Discord](https://discord.gg/KqswhpVgdU) to get in touch with us.
 
-## Installation from Source
+## Installation
 
 <details>
-  <summary>Build from Source</summary>
+  <summary>Build from source</summary>
 
 ### Install Rust and base dependency
+
 ```sh
-apt install -y build-essential libpq-dev libssl-dev pkg-config gcc libreadline-dev flex bison libxml2-dev libxslt-dev libxml2-utils xsltproc zlib1g-dev ccache clang
+sudo apt install -y build-essential libpq-dev libssl-dev pkg-config gcc libreadline-dev flex bison libxml2-dev libxslt-dev libxml2-utils xsltproc zlib1g-dev ccache clang git
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 ```
 
-### Install pgrx (tensorchord's fork)
+### Clone the Repository
+
+```sh
+git clone https://github.com/tensorchord/pgvecto.rs.git
+cd pgvecto.rs
+```
+
+### Install Postgresql and pgrx
+
 ```sh
+sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+sudo apt-get update
+sudo apt-get -y install libpq-dev postgresql-15 postgresql-server-dev-15
 cargo install cargo-pgrx --git https://github.com/tensorchord/pgrx.git --rev $(cat Cargo.toml | grep "pgrx =" | awk -F'rev = "' '{print $2}' | cut -d'"' -f1)
```

Comment:
Can we update to official pgrx beta version since the header file is included?

Reply:
@VoVAllen I'm not sure if we need include more in the future.